lib/bitpay/rest_connector.rb

```diff
@@ -8,9 +8,11 @@ def send_request(verb, path, facade: 'merchant', params: {}, token: nil)
       token ||= get_token(facade)
       case verb.upcase
       when "GET"
-        return get(path: path, token:token)
+        return get(path: path, token: token)
       when "POST"
         return post(path: path, token: token, params: params)
+      when "DELETE"
```

Comment:
Not sure we should add the "DELETE" branch here: we intend to deprecate this method.